catalog/acm/config-mgmt-csr.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  name: feature-membership-name # kpt-set: acm-mem-${cluster-name}
+  namespace: platform-namespace # kpt-set: ${platform-namespace}
+  annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/acm/v0.0.1
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${platform-project-id}
+spec:
+  projectRef:
+    external: platform-project-id # kpt-set: ${platform-project-id}
+  location: global
+  membershipRef:
+    name: membership-name # kpt-set: hub-mem-${cluster-name}
+  featureRef:
+    name: feature-name # kpt-set: feat-acm-${cluster-name}
+  configmanagement:
+    version: 1.9.0 #kpt-set: ${acm-version}
+    configSync:
+      sourceFormat: unstructured
+      git:
+        syncRepo: https://source.developers.google.com/p/{project_id}/r/{repo} # kpt-set: ${sync-repo}
+        syncBranch: main
+        policyDir: config/root # kpt-set: ${policy-dir}
+        syncWaitSecs: "10"
```

Comment:
I think we can remove this and let KCC default to `15`

Reply:
Remove this field as well